src/main.py

```diff
@@ -1,9 +1,25 @@
 from fastapi import FastAPI
 from routers.health import health_router
 from routers.mtcars import data_output
+from docs.documentation import description
 import uvicorn
 
-app = FastAPI()
+app = FastAPI(
+    # This info goes directly into /docs
+    title="RMI Web API poc",
+    # Description of API defined in docs/documentation.py for ease of reading
+    description=description,
```

Comment:
Similar to the comment below, I wonder if it might be easier to specify the `description` once in the `pyproject.toml`, and use that as our source of truth for all app info. Will save us maintenance burden if we are ever trying to keep those two things in sync.

Reply:
(e.g. copy the contents of `src/docs/documentation.py` to `pyproject.toml` under the `description` header, and load it up from there)